python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml

```
@@ -114,8 +114,8 @@ $defs:
                 items:
                   type: str
                 description: |
-                  Nodes is required to restrict configuration of BGP neighbors to certain nodes in the network.
-                  If not set the peer-group is created on devices which have a bgp_peer mapped to the corresponding peer_group.
+                  List of node names or fully matching regex to configure the peer group, e.g. 'DC1-BL[12]B'.
```

```suggestion
                  List of node names or fully matching regexes to configure the peer group, e.g. 'DC1-BL[12]B'.
```

Can we add a test with 2 regexes?

---

Added.